src/components/app/MenuButtons/Publish.js

```diff
@@ -120,39 +121,59 @@ class MenuButtonsPublishImpl extends React.PureComponent<PublishProps> {
         <form className="menuButtonsPublishContent" onSubmit={attemptToPublish}>
           <div className="menuButtonsPublishIcon" />
           <h1 className="menuButtonsPublishTitle">
-            {isRepublish
-              ? 'Re-upload Performance Profile'
-              : 'Share Performance Profile'}
+            {isRepublish ? (
+              <Localized id="MenuButtons--publish--reupload-performance-profile">
+                Re-upload Performance Profile
+              </Localized>
+            ) : (
+              <Localized id="MenuButtons--publish--share-performance-profile">
+                Share Performance Profile
+              </Localized>
+            )}
           </h1>
           <p className="menuButtonsPublishInfoDescription">
-            Upload your profile and make it accessible to anyone with the link.
-            {shouldSanitizeByDefault
-              ? ' By default, your personal data is removed.'
-              : ' This profile is from Firefox Nightly, so by default all information is included.'}
+            <Localized id="MenuButtons--publish--info-description">
+              Upload your profile and make it accessible to anyone with the
+              link.
+            </Localized>{' '}
+            {shouldSanitizeByDefault ? (
+              <Localized id="MenuButtons--publish--info-description-default">
+                By default, your personal data is removed.
+              </Localized>
+            ) : (
+              <Localized id="MenuButtons--publish--info-description-firefox-nightly">
+                This profile is from Firefox Nightly, so by default all
+                information is included.
+              </Localized>
+            )}
           </p>
-          <h3>Include additional data that may be identifiable</h3>
+          <h3>
+            <Localized id="MenuButtons--publish--include-additional-data">
+              Include additional data that may be identifiable
+            </Localized>
+          </h3>
           <div className="menuButtonsPublishDataChoices">
             {this._renderCheckbox(
               'includeHiddenThreads',
-              'Include hidden threads'
+              'MenuButtons--publish--renderCheckbox-label-hidden-threads'
             )}
             {this._renderCheckbox(
               'includeFullTimeRange',
-              'Include hidden time range'
+              'MenuButtons--publish--renderCheckbox-label-hidden-time'
             )}
             {this._renderCheckbox('includeScreenshots', 'Include screenshots')}
```

Comment:
ooops, looks like you missed this one :-)
![image](https://user-images.githubusercontent.com/454175/111479159-01e71200-8731-11eb-9660-3c21953a0af5.png)